bin/suitcss

```diff
@@ -65,7 +65,13 @@ program.parse(process.argv);
 
 var input = program.args[0] ? resolve(program.args[0]) : null;
 var output = program.args[1] ? resolve(program.args[1]) : null;
-var config = program.config ? require(resolve(program.config)) : null;
+var config;
+var configFile = program.config;
+if (configFile && configFile.endsWith('.js')) {
+  config = require(resolve(configFile));
+} else if (configFile) {
+  config = fs.readJsonSync(resolve(configFile));
+}
```

Comment:
Maybe we could put this into a function to reduce the duplication? Something like:

```js
var config = resolveConfig(program.config);

function resolveConfig(config) {
  if (!config) {return;}
  var configPath = resolve(configFile);
  var ext = path.extname(configFile);

  if (/js|json/.test(ext)) {
    return require(configPath);
  }

  return fs.readJsonSync(configFile);
}
```

Note I haven't tested the above, but it will allow us to leverage `require` for JSON files too and fix the ESLint issue.